demos/focused_api_test.py

```
@@ -8,29 +8,44 @@
 from pathlib import Path
 
-# Load .env file
-env_path = Path(__file__).parent / ".env"
-if env_path.exists():
-    with open(env_path) as f:
-        for line in f:
-            line = line.strip()
-            if line and not line.startswith('#') and '=' in line:
-                key, value = line.split('=', 1)
-                os.environ[key] = value
+def load_env():
```

Ruff (0.11.9)

Missing return type annotation for public function `load_env`

Add return type annotation: `None`

(ANN201)